python/src/PythonGreyscale.cc

```diff
@@ -0,0 +1,47 @@
+#include <boost/python/numpy.hpp>
+#include <iostream>
+#include <vector>
+#include "PythonGreyscale.h"
+#include "npp.h"
+#include "ipp.h"
+#include "Format.h"
+
+namespace np = boost::python::numpy;
+
+namespace lightdb::python {
+    shared_reference <LightField> PythonGreyscale::CPU::operator()(LightField &input) {
+        np::initialize();
+        const auto channels = 3u;
+        auto &data = dynamic_cast<lightdb::physical::CPUDecodedFrameData&>(input);
+        lightdb::physical::CPUDecodedFrameData output(data.configuration(), data.geometry());
+
+         for(auto& frame: data.frames()) {
+            Allocate(frame->height(), frame->width(), channels);
+            IppiSize size{static_cast<int>(frame->width()), static_cast<int>(frame->height())};
+            auto y_in = reinterpret_cast<const unsigned char*>(frame->data().data());
+            auto uv_in = y_in + frameSize_;
+            output.frames().emplace_back(LocalFrameReference::make<LocalFrame>(*frame, frame->data().size(), lightdb::video::Format::nv12()));
```

Comment:
Move NV12->RGB to a dedicated function in case we support other format conversions

Reply:
Maybe use frame->format() instead of hardcoding format?